products/messaging/frontend/Channels/messageChannelsLogic.tsx

```diff
@@ -0,0 +1,46 @@
+import { actions, connect, kea, path, reducers } from 'kea'
+import { integrationsLogic } from 'lib/integrations/integrationsLogic'
+
+import { IntegrationType } from '~/types'
+
+import type { messageChannelsLogicType } from './messageChannelsLogicType'
+
+export type ChannelType = 'email' | 'slack' | 'twilio' | 'webhook'
+
+export const messageChannelsLogic = kea<messageChannelsLogicType>([
+    path(['products', 'messaging', 'frontend', 'messageChannelsLogic']),
+    connect(() => ({
+        values: [integrationsLogic, ['integrations', 'integrationsLoading']],
+        actions: [integrationsLogic, ['deleteIntegration']],
+    })),
+    actions({
+        openNewChannelModal: (integration?: IntegrationType, channelType?: ChannelType) => ({
+            integration,
+            channelType,
+        }),
+        closeNewChannelModal: () => null,
```

Comment:
```suggestion
        closeNewChannelModal: true,
```
In case your actions take no arguments (e.g. loadUsers), just pass true.

https://keajs.org/docs/core/actions/